charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -159,6 +159,11 @@ data:
                   # to the airflow namespaces.
                   secretName: {{ template "houston.jwtCertificateSecret" . }}
             {{ end  }}
+            {{ if .Values.global.openshiftEnabled }}
+            securityContexts:
+              pod:
+                runAsNonRoot: true
+            {{ end  }}
```

Comment:
If this works for openshift, can we just use it as the standard config? Is there any good reason to run it without this setting when we are running in a non-openshift cluster?

Reply:
I was thinking on the same lines. We can keep changes agnostic to environoment as the standard config and openshift deployment specific config under the flag

Reply:
Agreed on Daniels point. I think the biggest shift is switching logging from FluentD to Vector as a default which would probably cause issues to existing customers.

Reply:
We should reduce the variety between openshift and non-openshift as much as possible. The more we reduce that diff, the less code we have to maintain, and the less things there are that can break, and the less QA testing needs to be done.